.travis.yml

```diff
@@ -25,6 +26,10 @@ matrix:
           env: SYMFONY_VERSION=2.7.x
         - php: 7.0
           env: SYMFONY_VERSION=2.8.x
+        - php: 7.0
+          env: SYMFONY_VERSION=3.0.x
```

Comment:
Symfony 3.0 has read EOM. Not sure you really want to run tests with this release. I'd rather use Symfony 3.2 instead.